CONTRIBUTING.md

```diff
@@ -92,10 +95,22 @@ Infrastructure engineers may want to look at the
 [`infrastructure` module's `README`](./infrastructure/README.md), which includes
 how to configure your local machine or a virtual machine as a video bridge host.
 
+### 2.2.1 Developing Mailers.
+
+Set [SMTP\_\* environment variables to configure Action Mailer](.env.example).
+
+To preview mailers, visit http://localhost:3000/rails/mailers/
```

Comment:
Part of me thinks this belongs in `docs/mailers.md` and should include a link to the Rails guide on ActionMailer.

Reply:
Added a line beneath this one to link to said Rails Guide.